module/mock/job_consumer.go

```diff
@@ -23,14 +23,14 @@ func (_m *JobConsumer) NotifyJobIsDone(_a0 module.JobID) {
 }
 
 // Size provides a mock function with given fields:
-func (_m *JobConsumer) Size() int {
```

Comment:
Not sure why this file is committed. This change was made by re-generating the mocks. Should I revert the change or remove this file entirely?

Reply:
it should be fine, it happens from time to time that a mock was not commited, I think we should keep this change.